### PR TITLE
Encode public_keys array for threshold conditions

### PR DESCRIPTION
**Pull request by libscott**
Previously wasn't making `output.public_keys` array for threshold conditions

### CODE REVIEW

File: src/transaction/makeOutput.js [hunk 1]
```diff
@@ -10,10 +10,20 @@ export default function makeOutput(condition, amount = '1') {
     if (typeof amount !== 'string') {
         throw new TypeError('`amount` must be of type string')
     }
+    const publicKeys = []
+    const getPublicKeys = details => {
+        if (details.type === 'ed25519-sha-256') {
+            if (publicKeys.indexOf(details.public_key) === -1) {
```
Comment (TimDaub) [inline src/transaction/makeOutput.js]: There is this seksiness now: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Comment (libscott) [inline src/transaction/makeOutput.js]: IE and Android don't support it, I figure stick to `indexOf` for a while.

Comment (TimDaub) [inline src/transaction/makeOutput.js]: We use webpack and shims so you can use it

Comment (libscott) [inline src/transaction/makeOutput.js]: okay

**Pull request merged**